source/algorithms.tex

```diff
@@ -8088,9 +8132,11 @@
 For the first form, type \tcode{T} shall be \oldconcept{LessThanComparable}.
 
 \pnum
-\returns
 Let \tcode{X} be the return type.
-Returns \tcode{X\{x, y\}},
+
+\pnum
+\returns
+\tcode{X\{x, y\}},
```

Comment:
Note the missing braces in the code.
Before:
![1571765957](https://user-images.githubusercontent.com/21071787/67313716-6c126d80-f4d1-11e9-868b-927d1e4407cc.png)
After:
![1571765920](https://user-images.githubusercontent.com/21071787/67313722-6e74c780-f4d1-11e9-8ee7-883457361529.png)